src/action_cc.cc

```diff
@@ -190,13 +190,28 @@ NM_ACTION_(nickel_setting) {
 
     bool v = false;
 
-    if (!strcmp(arg, "invert") || !strcmp(arg, "screenshots")) {
+    char *tmp1 = strdupa(arg); // strsep and strtrim will modify it
+    char *arg1 = strtrim(strsep(&tmp1, ":"));
+    char *arg2 = strtrim(tmp1);
+
+    if (arg2) {
+        // note: v gets inverted when setting the setting below
+        if (!strcmp(arg2, "true") || !strcmp(arg2, "enabled") || !strcmp(arg2, "1")) {
```

Comment:
I'd add `yes`/`no` & `on`/`off` to the list ;).

Reply:
Done.